docs/configuration-reference/backend/backend.md

```diff
@@ -0,0 +1,77 @@
+# Backend configuration guide
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Argument reference](#argument-reference)
+
+## Introduction
+
+Lokomotive installer supports local or remote backend [S3 only] for storing terraform state.
```

Comment:
```suggestion
Lokomotive installer supports local or remote backend [S3 only] for storing Terraform state.
```

Reply:
Hm, phrase "Lokomotive installer" sounds weird to me. Should we just say `Lokomotive` or just `lokoctl` maybe?